submit_here/click_me_to_edit/include_for_all.txt

```diff
@@ -0,0 +1,129 @@
+www.loamarine.com
+elinajaguar.com
+3psil0n.fr
+tangible-drink.surge.sh
+womanflirting.life
+bonustop-price.life
+allprizeshub.life
+greatbonushere.top
+prizes-topwin.life
+a.crystalcraft.top
+logsmetrics.com
+webdatatrace.com
+marybskitchen.com
+prom-gg.com
+go.clicksme.org
+machinetext.org
+getquery.org

```

Comment:
Maybe you should make a check for active vs inactive

```bash
:~$ drill -T getquery.org
.       518400  IN      NS      g.root-servers.net.
.       518400  IN      NS      a.root-servers.net.
.       518400  IN      NS      e.root-servers.net.
.       518400  IN      NS      k.root-servers.net.
.       518400  IN      NS      m.root-servers.net.
.       518400  IN      NS      b.root-servers.net.
.       518400  IN      NS      i.root-servers.net.
.       518400  IN      NS      c.root-servers.net.
.       518400  IN      NS      d.root-servers.net.
.       518400  IN      NS      f.root-servers.net.
.       518400  IN      NS      h.root-servers.net.
.       518400  IN      NS      j.root-servers.net.
.       518400  IN      NS      l.root-servers.net.
org.    172800  IN      NS      a2.org.afilias-nst.info.
org.    172800  IN      NS      b0.org.afilias-nst.org.
org.    172800  IN      NS      d0.org.afilias-nst.org.
org.    172800  IN      NS      c0.org.afilias-nst.info.
org.    172800  IN      NS      a0.org.afilias-nst.info.
org.    172800  IN      NS      b2.org.afilias-nst.org.
org.    3600    IN      SOA     a0.org.afilias-nst.info. hostmaster.donuts.email. 1706610321 7200 900 1209600 3600
```

Reply:
I guess I assumed there is logic within the daily build process that culls old domains.  Do you know if this is not the case?

Reply:
Hmm.  Looks like there is no logic to test and remove unnecessary hosts from the files.  That would be an excellent enhancement/pull request I would think.

Reply:
This repo do relay on third party lists, which means it do not test and/or remove unnecessary records only very few projects like @mypdns and a few other does this regularly to keep the source as small as possible. (Some of my lists takes up to a week for testing)